recipes/chewbbaca/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "chewBBACA" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/B-UMMI/chewBBACA/archive/{{ version }}.tar.gz
+  sha256: c4a34c3ecd780cacfdabbb116cba36ced11cba36fefdf4ca9bea9ecd3c918d71
+
+build:
+  skip: True  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy 1.13.1
```

Comment:
Is this strict pinning here needed?
Also for biopython and prodigal ...

Reply:
I've done my best to match the versions listed on the [chewBBACA repo](https://github.com/B-UMMI/chewBBACA) and in the `requirements.txt` file